src/angular-app/languageforge/lexicon/editor/field/dc-audio.component.ts

```diff
@@ -115,6 +115,9 @@ export class FieldAudioController implements angular.IController {
             this.dcFilename = response.data.data.fileName;
             this.showAudioUpload = false;
             this.notice.push(this.notice.SUCCESS, 'File uploaded successfully.');
+            if(response.data.data.fileSize > 1000000){ //1 MB file size limit 2022-09
```

Comment:
I'm good with 1 million bytes here, even though a megabyte likely refers to [1024 x 1024 = 1,048,576 bytes](https://www.backblaze.com/blog/how-many-bytes-are-in-a-megabyte-really/).  I still get confused by this and am not 100% sure which one the Chorus 1MB limit is really referring to.